end_to_end_tests/run-e2e-all-scopes.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/sh
```

Comment:
I have no idea about this backend scripts if they are needed or not. Or are they really part of this pull request?

Reply:
Removed this file from the branch as it contained our backend credentials